lib/src/platform/src/push_notification_events_internal.dart

```diff
@@ -5,20 +5,25 @@ import 'dart:ui';
 import 'package:ably_flutter/ably_flutter.dart';
 import 'package:ably_flutter/src/platform/platform_internal.dart';
 
+/// @nodoc
 /// package-private implementation of [PushNotificationEvents]
 class PushNotificationEventsInternal implements PushNotificationEvents {
+  /// @nodoc
   /// Invoked when pushOpenSettingsFor platform method is called
```

Comment:
All comments in this file could be "upgraded" to sentences - starting with a capital letter and ending with the dot if we want to

Reply:
Done in 19b0814c30acaf767beee3ed6eecd362b672f012